.github/workflows/changelog_verification.yml

```diff
@@ -0,0 +1,40 @@
+name: Changelog Verification
+on:
+  pull_request:
+    branches:
+      - '**/release-**'
+  push:
+    branches:
+      - '**/release-**'
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+*'
```

Comment:
Are you sure the `.` character doesn't need to be escaped? pls check with a tag like `v3i1+8`